engine/function/src/templates/Numeric.ftl

```diff
@@ -1606,10 +1612,18 @@ public class Numeric {
         try ( final ${pt.vectorIterator} vi = values.iterator() ) {
             result[0] = vi.${pt.iteratorNext}();
             int i = 1;
-    
+
             while (vi.hasNext()) {
                 final ${pt.primitive} v = vi.${pt.iteratorNext}();
-                result[i] = compare(result[i-1], v) > 0 ? result[i-1] : v;
+
+                if (isNull(result[i - 1])) {
+                    result[i] = v;
+                } else if (isNull(v)) {
+                    result[i] = result[i - 1];
+                } else {
+                    result[i] = (${pt.primitive})Math.max(result[i - 1],  v);
```

Comment:
Can we remove the extra space before v at the end

Reply:
```suggestion
                    result[i] = (${pt.primitive})Math.max(result[i - 1], v);
```